repo2docker/buildpacks/julia.py

```diff
@@ -12,6 +12,10 @@ class JuliaBuildPack(CondaBuildPack):
     See https://github.com/JuliaPy/PyCall.jl/issues/410
 
     """
+
+    # REQUIRE can't refer to local files, can it?
+    assemble_from_subset = True
```

Comment:
Worth checking with #393 and the new way of specifying dependencies in Julia.

Reply:
Good catch! I think it will affect Julia with the project stuff.